### ISSUE TITLE
Fix BrowserStack tests

### ISSUE THREAD

**Issue by shawnbuso** [googleads/videojs-ima#616]
The BrowserStack tests are super flaky and fail at random with every run. They're currently disabled - let's fix them and re-enable them.

**Comment by mysuf** [comment]
One recommendation based on my experiences:
I had to add player.ready to tests as videojs is growing and it takes too long to load to dispatch webdriver click (with v6.10.0 firefox always failed in random tests). This fixed it for me:

**ads.js**
```
player.ready(function() {
  var log = document.getElementById('log');
  log.innerHTML += "ready<br>"; 
});
```

**basic.js:**
```
let log = await driver.findElement(By.id('log'));
await driver.wait(until.elementTextContains(log, 'ready'), 10000);
await driver.findElement(By.id('content_video')).click();
....
```

**Comment by ypavlotsky** [comment]
Fixed with #657
